data/mods/blazemod/blaze_override.json

```diff
@@ -1,139 +1,348 @@
 [
-	{
+     {
+        "type" : "construction",
+        "description" : "Start Vehicle Construction", "//": "Rest of vehicle construction done in the vehicle construction menu", 
+        "category" : "OTHER",
+        "difficulty" : 0,
+        "time" : 10,
+        "components" : [
+            [
+                [ "frame", 1 ],
+                [ "hdframe", 1 ],
+                [ "frame_wood", 1 ],
+                [ "xlframe", 1 ],
+                [ "frame_wood_light", 1 ],
+                [ "foldwoodframe", 1 ],
+                [ "foldframe", 1 ],
+                [ "gloople", 1],
+                [ "oozle", 1],
+                [ "gray", 1],
+                [ "diamondframe", 1]
+            ]
+        ],
+        "pre_special" : "check_empty",
+        "post_special" : "done_vehicle"
+    },{
```

Comment:
Could someone vastly more knowledgeable than me check if this override to "Start Vehicle Construction" still causes player constructed initial parts to instant delete?
Also I seem to remember a recent commit that rendered this change obsolete with the base version of "Start Vehicle Construction" now using anything with the initial_part flag.